modules/exploits/windows/fileformat/shaper_pdf_bof.rb

```diff
@@ -0,0 +1,132 @@
+##
+# This module requires Metabuffer: http://metabuffer.com/download
+# Current source: https://github.com/rapid7/metabuffer-framework
```

Comment:
I don't know from where you got Metabuffer but its Metasploit. All the Metabuffer strings need to be changed to Metasploit.